docs/adr/0001-actors.rst

```diff
@@ -0,0 +1,110 @@
+ADR 0001 - Actors
+=================
+
+Status
+------
+
+Proposed
+
+
+Context
+-------
+
+There are different kinds of actors that need to interact with Marsha.
+
+First we have the people managing a Marsha instance.
+
+Then we have people linking their own website (this website is a "consumer site") to a Marsha instance, to host videos.
+
+These consumer sites can host many customers. We call these organizations. And these organizations have managers that can administrate some things about authors, video sharing between authors...
+
+Next we have the video authors, belonging the organizations.
+
+And finally we have the users coming to watch videos.
+
+
+Decision
+--------
+
+Let's separate those 5 actors / roles:
+
+"staff"
++++++++
+
+Purpose
+~~~~~~~
+
+To manage a Marsha instance
+
+Implementation
+~~~~~~~~~~~~~~
+
+These are simply instances of the Django ``User`` model, with the flag ``is_staff`` set to ``True``.
+
+
+"admins"
+++++++++
+
+Purpose
+~~~~~~~
+
+To manage the link between a consumer site and a Marsha instance, and the organizations of this consumer site on the instance.
+
+Implementation
+~~~~~~~~~~~~~~
+
+To represent a consumer site on an Marsha instance, we have a ``ConsumerSite`` Django model. With a ``ManyToMany`` link to the ``User`` model, named ``admins`` (not a single admin, to avoid having no admin if the only one existing is not available anymore)).
+
+
+"managers"
+++++++++++
+
+Purpose
+~~~~~~~
+
+To manage the authors in the organization (an organization could be present on many consumer sites). And allow videos to be private to authors or public for all authors.
+
+Implementation
+~~~~~~~~~~~~~~
+
+To represent an organization on a Marsha instance, we have a ``Organization`` Django model. With a ``ManyToMany` link to the ``User`` model, named ``managers``.
```

Comment:
Will the `Organization` model be related to the `auth.Group` model to handle permissions? If yes, how?

Reply:
We'll see. I voluntarily didn't talk about precise rights implementation in this document.